docs/src/main/sphinx/admin/fault-tolerant-execution.rst

```diff
@@ -352,6 +397,20 @@ does not have to be in AWS, but can be any S3-compatible storage system.
     exchange.s3.aws-access-key=example-access-key
     exchange.s3.aws-secret-key=example-secret-key
 
+You can configure multiple S3 directories for the exchange manager to use,
+distributing spooled data between them. If a query fails with the error message
```

Comment:
reword .. you are not doing any distributing .. the system does that

Reply:
How about "for the exchange manager to distribute spooled data between each directory"

Reply:
"between each directory" -> "among buckets"?